ci/run_easymesh_cert.sh

```diff
@@ -71,7 +71,7 @@ main() {
     info "Tests to run: $TESTS"
 
     info "download latest ipk from branch $BRANCH"
-    run "$TOOLS_PATH"/download_ipk.sh --branch "$BRANCH"
+    run "$TOOLS_PATH"/download_ipk.sh --quite --branch "$BRANCH" "$VERBOSE_OPT"
```

Comment:
quite -> quiet - but that option is not actually implemented in download_ipk.sh.

VERBOSE_OPT shouldn't be quoted, otherwise you're passing an extra empty argument when verbose is not set. So with shellcheck's suggestion:
```suggestion
    run "$TOOLS_PATH"/download_ipk.sh --branch "$BRANCH" ${VERBOSE:+ -v}
```